pkg/apis/core/validation/validation.go

```diff
@@ -8595,3 +8620,11 @@ func validateImageVolumeSource(imageVolume *core.ImageVolumeSource, fldPath *fie
 	allErrs = append(allErrs, validatePullPolicy(imageVolume.PullPolicy, fldPath.Child("pullPolicy"))...)
 	return allErrs
 }
+
+// isRestartableInitContainer returns true if the container has ContainerRestartPolicyAlways.
+func isRestartableInitContainer(initContainer *core.Container) bool {
```

Comment:
I am not able to use `podutil.IsRestartableInitContainer` since that function is defined in pkg/api/pod/util.go where we are importing `apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"` (current package) leading to import cycle error.